compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -560,6 +560,15 @@ def PadDynamicAllocPass :
 def MathTransformPass :
     Pass<"iree-codegen-math-transform", ""> {
   let summary = "Apply math ops transformations: approximations, rewrites to other math ops, operand casts.";
+  let options = [
+    Option<"hasFastExp", "has-fast-exp", "bool",
+            /*default=*/"false",
+           "Enable device-lib implementations for operations that rely on fast exp">
+  ];
+  let dependentDialects = [
```

Comment:
Introducting this option is a departure from how this pass is currently doing all target-specialization from the target attribute, eschewing the need for such options. How about instead computing `bool hasFastExp = isROCMBackend(target)`  (there is much more information in `target` if we need this to become more specific to some target hardware).